src/runners/handlers/slack.py

```diff
@@ -65,37 +65,61 @@ def handle(
 ):
     slack_token_ct = slack_api_token or api_token
     slack_token = vault.decrypt_if_encrypted(slack_token_ct)
-
     sc = SlackClient(slack_token)
 
     # otherwise we will retrieve email from assignee and use it to identify Slack user
     # Slack user id will be assigned as a channel
 
-    title = alert['TITLE']
-
+    # title = alert['TITLE']
+    title = 'title'
     if recipient_email is not None:
-        result = sc.api_call("users.lookupByEmail", email=recipient_email)
-
-        # log.info(f'Slack user info for {email}', result)
-
-        if result['ok'] is True and 'error' not in result:
-            user = result['user']
-            userid = user['id']
-        else:
-            raise RuntimeError(f'no Slack user for email {recipient_email}')
-
+        if isinstance(recipient_email, str):
+                user = sc.api_call("users.lookupByEmail", email=recipient_email)
+                print(user)
+                if not user['ok']:
+                    log.error(f'Cannot identify Slack user for email {recipient_email}')
+                    return None
+                    
+                else:
+                    userid = user['user']['id']
+                    result = sc.api_call("conversations.open", users=userid)
+                    if result['ok']:
+                        channel_id = result['channel']['id']
+                    else:
+                        log.error(f'Error ocurred while opening conversation channel')
+                        return None
+                        
+        elif isinstance(recipient_email, list):
+            users = []
+            for email in recipient_email:
+                user = sc.api_call("users.lookupByEmail", email=email)
+                print(user)
```

Comment:
remove vestigial print statements. they may seem harmless, but the can actually increase the sensitivity classification of logs, which ideally should not contain employee PII. if you raise or return this value, the error will be stored in Snowflake.